### ISSUE TITLE
suggestion: swapper malus

### ISSUE THREAD

**Issue by lvictorino** [Elao/curvytron#107]
I'm not sure to understand what the color swapper bonus does... but what about and actual swapper? Instead of a simple color changing players actually warps at another player position?

**Comment by Tom32i** [comment]
Yeah, the color bonus is my least favourite. Maybe we can get rid of it.

The problem with randomly changing players positions is you probably need to pause the game for a few secondes to let the players finds their "new" position on the map. And I that would completely kill the groove.

**Comment by lvictorino** [comment]
not necessarily. If you give them enough feedback to know when or where they will swap it should be just enough disturbing to create "fun chaos".

**Comment by jpetitcolas** [comment]
Good idea. For instance, when taking bonus, all players flashes with their new colors during one or two seconds, to let them see where they are going to be teleported.
